paasta_tools/cli/cmds/spark_run.py

```diff
@@ -1102,8 +1102,28 @@ def _validate_pool(args, system_paasta_config):
 
 
 def _get_k8s_url_for_cluster(cluster: str) -> Optional[str]:
+    """
+    Annoyingly, there's two layers of aliases: one to figure out what
+    k8s server url to use (this one) and another to figure out what
+    soaconfigs filename to use ;_;
+
+    This exists so that we can map something like `--cluster pnw-devc`
+    into spark-pnw-devc's k8s apiserver url without needing to update
+    any soaconfigs/alter folk's muscle memory.
+
+    Ideally we can get rid of this entirely once spark-run reads soaconfigs
+    in a manner more cloesly aligned to what we do with other paasta workloads
```

Comment:
tiny typo (not a blocker)

```suggestion
    in a manner more closely aligned to what we do with other paasta workloads
```